homeassistant/components/aws/image_processing.py

```diff
@@ -83,7 +85,8 @@ def save_image(self, image, entity_name, objects, directory, save_timestamp=Fals
             text="X",
             fill=box_colour,
         )
-    filename = get_valid_filename(entity_name).lower()
+    filename = await get_valid_filename(entity_name)
```

Comment:
could just await get_valid_filename(entity_name).lower()

Reply:
That would attempt to apply `.lower()` on the coroutine object rather than the result itself. Could possibly wrap a helper function but I think it's better for readability to run `.lower()` on the result 😄

Reply:
ah its coroutine, fair enough